### ISSUE TITLE
123

### ISSUE THREAD

**Issue by eclee25** [HopkinsIDD/cholera-database#44]

**Comment by scottyaz** [comment]
@eclee25 can you confirm you agree that the phantom observations I created should also be primary?

**Comment by eclee25** [comment]
Yep - any phantom observations with only space and time stratification should also be primary.

**Comment by eclee25** [comment]
It seems like there is a duplicate phantom observation for Abidjan?
